### DISCUSSION TITLE
Wrong TOTP code

### DISCUSSION THREAD

**Discussion by thewalkerz** [keepassxreboot/keepassxc#9275] [Q&A] [answered]
Hi Keepass community,

I would like to first thank you for the great work on this password manager, I use it everyday, really love it.

I have a strange problem with the TOTP, when I configure a new one it gives me an erroneous code, I tried it with another totp app and this one works for me, but not with keepassxc, using the rfc 6238 standard in both. Am I doing something wrong?

This only happened to me on my job's laptop on my personal computer works great, I'm using portable version!

**Comment by droidmonkey** [comment, marked as answer]
Sync your computer time
